src/util/mpir_pmi.c

```diff
@@ -67,6 +67,14 @@
 === END_MPI_T_CVAR_INFO_BLOCK ===
 */
 
+#ifdef USE_PMI2_SLURM
+#define INFO_TYPE struct MPID_Info
```

Comment:
In Slurm's pmi2.h they also define
```
#define PMI2U_Info MPID_Info
```
would it be better to use PMI2U_Info in case they change the underlying struct at some point?

Reply:
I neglected that. Yeah, `PMI2U_Info` is much better than `MPID_Info`.